cinema/serializers.py

```diff
@@ -1,24 +1,113 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
 class MovieSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
     title = serializers.CharField(max_length=255)
     description = serializers.CharField()
     duration = serializers.IntegerField()
+    actors = serializers.PrimaryKeyRelatedField(
+        queryset=Actor.objects.all(), many=True
+    )
+    genres = serializers.PrimaryKeyRelatedField(
+        queryset=Genre.objects.all(), many=True
+    )
```

Comment:
you were not supposed to do so in **_this_** task according to the description
![image](https://github.com/mate-academy/py-api-views/assets/108356354/6ab9eabb-5fd5-41e3-b607-1fe27907c34e)